src/rating/rating.ts

```diff
@@ -13,26 +13,18 @@ import {Component, Input, Output, NgFor, EventEmitter, OnInit} from 'angular2/an
   directives: [NgFor]
 })
 export class NgbRating implements OnInit {
-  @Input() private max: number = 10;
-  @Input() private rate: number;
-  @Input() private readonly: boolean;
-  @Output() private rateChange: EventEmitter = new EventEmitter();
-  @Output() private hover: EventEmitter = new EventEmitter();
-  @Output() private leave: EventEmitter = new EventEmitter();
-  private oldRate: number;
```

Comment:
You quoted the wrong line. But we were 2vs1, you and I decided to go Array<foo>; and Pawel decided to go number[].